src/views/download-view.js

```diff
@@ -361,6 +361,27 @@ class DownloadView extends PolymerElement {
         }, 100);
     }
 
+    compareVersions(a, b) {
+        return this.compareParsedVersions(this.parseVersion(a), this.parseVersion(b));
+    }
+
```

Comment:
`v1.1.10` is more up to date than `v1.1.10-hotfix-asn-element`. Is that on purpose? `v.1.2.7-pre-release` would also be more up to date than `v.1.2.7` (if we would have had one^^) which should not be the case